deploy/kuberhealthy-prometheus-operator.yaml

```diff
@@ -426,7 +426,7 @@ spec:
         spec:
           containers:
             - name: check-reaper
-              image: quay.io/comcast/check-reaper:1.1.0
+              image: quay.io/comcast/check-reaper:1.0.0
```

Comment:
This looks wrong.  It dropped from `quay.io/comcast/check-reaper:1.1.0` to `quay.io/comcast/check-reaper:1.0.0`.